File: Java/1400. Construct K Palindrome Strings.java
class Solution {
    public boolean canConstruct(String s, int k) {
        if(s.length()<k){
            return false;
        }
        int[] arr = new int[26];
        for(int i = 0; i<s.length(); i++){
            arr[s.charAt(i)-'a']++;
        }
        int res=0;
        for(int i = 0; i<26; i++){
            if(arr[i]%2==1){
                res++;
            }
        }
        return res<=k;
    }
}

File: JavaScript/1400. Construct K Palindrome Strings.js
/**
 * @param {string} s
 * @param {number} k
 * @return {boolean}
 */
var canConstruct = function(s, k) {
    if(s.length<k){
        return false;
    }
    var arr = new Array(26).fill(0);
    var a = 'a'.charCodeAt(0);
    for(var i = 0; i<s.length; i++){
        arr[s.charCodeAt(i)-a]++;
    }
    var res=0;
    for(var i = 0; i<26; i++){
        if(arr[i]%2==1){
            res++;
        }
    }
    return res<=k;
};
